DQM/SiTrackerPhase2/interface/TrackerPhase2DQMUtil.h

```diff
@@ -0,0 +1,105 @@
+#ifndef _Validation_SiTrackerPhase2V_Phase2TrackierValidationUtil_h
+#define _Validation_SiTrackerPhase2V_Phase2TrackierValidationUtil_h
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include <string>
+#include <sstream>
+
+namespace Phase2TkUtil {
+
+  std::string getITHistoId(uint32_t det_id, const TrackerTopology* tTopo) {
```

Comment:
@sroychow am I wrong or this (and the other method `getOTHistoId` below, can be used as a drop-in replacement here:

https://github.com/cms-sw/cmssw/blob/9e912a119e2bf295939ef033dbc5fb485df778c9/Validation/SiTrackerPhase2V/plugins/Phase2TrackerValidateDigi.cc#L1073

and here:

https://github.com/cms-sw/cmssw/blob/473501c5bfcf807d6aaa022061d0d584c703a406/DQM/SiTrackerPhase2/plugins/Phase2TrackerMonitorDigi.cc#L711

?

Reply:
@mmusich yes this is intended to replace all getHistoId calls in the digi module as well.

Reply:
thanks. So this will be done in another separate PR?